### PR TITLE
Add ASP.NET Core quickstart

### PR DESCRIPTION
**Pull request by brendanzagaeski**
## Purpose

Add an ASP.NET Core quickstart that combines the use of Secret Manager as shown in the .NET Core quickstart with the ASP.NET quickstart.

To view these changes in the context of the quickstart articles, see the
corresponding pull request for the docs repo:
https://github.com/MicrosoftDocs/azure-docs-pr/pull/152899

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: quickstart/aspnet-core/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+name: 'Quickstart: Use Azure Cache for Redis with an ASP.NET Core web app'
+description: Learn how to use an ASP.NET Core web application to connect to Azure Cache for Redis to store and retrieve data from the cache.
+products:
+- azure
+- aspnet-core
+- azure-cache-redis
+---
+# Quickstart: Use Azure Cache for Redis with an ASP.NET Core web app
+
+This sample shows you how to use an ASP.NET Core web application to connect to Azure Cache for Redis to store and retrieve data from the cache. See the [accompanying article](https://docs.microsoft.com/azure/azure-cache-for-redis/cache-web-app-aspnet-core-howto) on the documentation site for details, including best practices and how to create the sample code from scratch.
```
Comment (brendanzagaeski) [inline quickstart/aspnet-core/README.md:14]: Note that the accompanying article on docs.microsoft.com does not yet exist. I'll add a link for a corresponding PR for that soon.

**Comment by philon-msft** [comment]
Are we also planning an ASP.NET quickstart that uses Microsoft.Extensions.Caching.StackExchangeRedis.RedisCache for distributed caching (https://docs.microsoft.com/en-us/aspnet/core/performance/caching/distributed?view=aspnetcore-5.0)? And session state?

**Review by philon-msft — APPROVED** [review]

**Comment by brendanzagaeski** [comment]
> Are we also planning an ASP.NET quickstart that uses Microsoft.Extensions.Caching.StackExchangeRedis.RedisCache for distributed caching (https://docs.microsoft.com/en-us/aspnet/core/performance/caching/distributed?view=aspnetcore-5.0)? And session state?

I think that's a good idea! I'll make sure I follow up on that this week and add items to track it.

**Comment by brendanzagaeski** [comment]
**Changes in the latest force-push:**

- Add a couple small improvements I noticed while working on the [corresponding docs.microsoft.com article](https://github.com/MicrosoftDocs/azure-docs-pr/pull/152899).
- Update the format of the files so they all consistently end with a trailing new line and use no byte order marks.

**Pull request merged**